cmd/snap/cmd_known.go

```diff
@@ -52,7 +55,12 @@ shown must also have the specified headers matching the provided values.
 func init() {
 	addCommand("known", shortKnownHelp, longKnownHelp, func() flags.Commander {
 		return &cmdKnown{}
-	}, nil, []argDesc{
+	}, map[string]string{
+		// TRANSLATORS: This should not start with a lowercase letter.
+		"remote": i18n.G("Query the store for the assertion via snapd"),
+		// TRANSLATORS: This should not start with a lowercase letter.
+		"direct": i18n.G("Query the store directly for the assertion"),
```

Comment:
... (cont'd)
```suggestion
		"direct": i18n.G("Query the store for the assertion, without attempting to go via snapd"),
```
 and then in the description we add a small explanation of the difference (we can work on that later though)